apps/server/.env.example

```
@@ -53,8 +53,16 @@ AWS_S3_PUBLIC_URL=http://localhost:29090/amped-bio
 
 CAPTCHA_SECRET_KEY=your_CAPTCHA_SECRET_KEY_here
 
-GOOGLE_CLIENT_SECRET=your_google_client_secret_here
-GOOGLE_CLIENT_ID=your_google_client_id_here
-
+ GOOGLE_CLIENT_SECRET=your_google_client_secret_here
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)